regolith/helpers/l_projectahelper.py

```diff
@@ -118,13 +123,17 @@ def sout(self):
         end_projecta = []
         grouped_projecta = {}
         if rc.lead and rc.person:
-            raise RuntimeError(f"please specify either lead or person, not both")
+            raise RuntimeError(
+                f"please specify either lead or person, not both")
         for projectum in collection:
-            if rc.all and projectum.get('status') != "finished":
-                projecta.append(projectum)
-                continue
             if isinstance(projectum.get('group_members'), str):
                 projectum['group_members'] = [projectum.get('group_members')]
+            if rc.all and (projectum.get('status') == True):
+                projecta.append(projectum)
+                continue
+            if rc.current and (projectum.get('status') == "proposed" or "started"):
```

Comment:
I think this will be hard to maintain.  I suggest that we make lists at the top with relevant things in them such as 
```
ACTIVE_STATI = ["proposed", "started"]
INACTIVE_STATI = [...]
```
and then say 
```
if rc.current and projectum.get('status') in ACTIVE_STATI:
...
```
Also, no need for those parens which have a different meaning in coding than in math (you just made a tuple there but luckily it has the desired behavior).   
Rather than littering our code with these tests, I think it is better if we make the projectum status to be required in the schema, so I prefer if you remove the ``projectum.get('status') == True`` tests everywhere (in this case the code crashing is a feature as it forces us to make the database compliant)